tests/test_Pipelines/test_skypy_pipeline.py

```diff
@@ -8,6 +9,40 @@ def setup_method(self):
         sky_area = Quantity(value=0.05, unit="deg2")
         self.pipeline = SkyPyPipeline(skypy_config=None, sky_area=sky_area)
 
+    def test_cosmology_initialization(self):
+        galaxy_cosmo0 = LambdaCDM(H0=70, Om0=0.01, Ode0=0.99)
+        pipeline0 = SkyPyPipeline(cosmo=galaxy_cosmo0)
+        blue_galaxies = pipeline0.blue_galaxies
+        red_galaxies = pipeline0.red_galaxies
+        cosmo0_blue_number = len(blue_galaxies)
+        cosmo0_red_number = len(red_galaxies)
+
+        # LambdaCDM model
+        cosmo1 = LambdaCDM(H0=70, Om0=0.3, Ode0=0.7)
+        pipeline1 = SkyPyPipeline(cosmo=cosmo1)
+        blue_galaxies = pipeline1.blue_galaxies
+        red_galaxies = pipeline1.red_galaxies
+        assert len(blue_galaxies) * 2 < cosmo0_blue_number
```

Comment:
this current test does not show that the number of objects with cosmology1 is different than the one with cosmology0